core/src/main/scala/org/apache/spark/h2o/utils/H2OContextUtils.scala

```diff
@@ -51,6 +54,60 @@ private[spark] trait H2OContextUtils extends Logging {
     }
   }
 
+  private def isTcpPortAvailable(port: Int): Boolean = {
+    try {
+      val serverSocket = new ServerSocket()
+
+      serverSocket.setReuseAddress(false);
+      serverSocket.bind(new InetSocketAddress(InetAddress.getByName(SparkEnv.get.blockManager.blockManagerId.host), port), 1)
+      serverSocket.close()
```

Comment:
Shouldn't `serverSocket.close()` be in the `finally` statement?

Reply:
Yup!